gtfs-netex-test/gtfs_show_map.py

```diff
@@ -38,7 +38,14 @@ def main(gtfs_zip_file,map_file):
         route_stop_coords=[]
         for astop in route_stops:
             x=df_stops[df_stops['stop_id']==str(astop)]
-            route_stop_coords.append([x['stop_lat'].values[0],x['stop_lon'].values[0]])
+            if not(len(x['stop_lat'])==0 or len(x['stop_lon'])==0):  # fix for non-existing coordinates
+                if not(x['stop_lat'].values[0]==0): # fix for coordinates set to the equator (also bad in our case)
+                    # the route only for reasonable set stops.
+                    route_stop_coords.append([x['stop_lat'].values[0],x['stop_lon'].values[0]])
```

Comment:
This should come from the shape. There is code available that matches this.

Reply:
@david1234 can update this. He is working on this.  In some cases there are no shapes, then  the stops should be used to create a pseudo-shape for viewing. Probably we should do this as a parameter